### PR TITLE
Mayhem Fuzzing

### PR DESCRIPTION
**Pull request by shreyasphx**
Added fuzzing integration with Mayhem

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+project: openml
+target: openml
+
+cmds:
```
Comment (xansec) [inline Mayhemfile:5]: Please either use only a single command, or separate this Mayhemfile into multiple Mayhemfiles as discussed here: https://community.forallsecure.com/t/error-the-mayhem-for-code-scan-detected-the-mayhem-run-for-your-target-was-unsuccessful-and-multiple-docker-files-in-the-repository/350/5

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+project: openml
+target: openml
```
Comment (xansec) [inline Mayhemfile:3]: Please make sure that the target name matches the target binary. So, if you were fuzzing `welcome_decode`, your target name should be `welcome_decode`

**Review by xansec — CHANGES_REQUESTED** [review]

**Comment by shreyasphx** [comment]
Reduced to 1 fuzz target in the mayhem file

**Comment by xansec** [comment]
LGTM!

**Pull request merged**